chain/client/src/client_actor.rs

```diff
@@ -1107,11 +1114,13 @@ impl ClientActor {
         });
     }
 
+    /// Call important functions of client, like running single step of state sync or checking if we
+    /// can produce a block.
+    ///
+    /// It is called during processing Actix message, and with at most 1 second period in `schedule_triggers`.
+    /// Returns the delay before the next invocation which is a minimum of all durations between
+    /// time of next trigger call and current time, and 1 second.
```

Comment:
I think the at most 1 second part is a bit confusing. Maybe change it to 

Check if the scheduled time of any "triggers" has passed, and if so, call the trigger. Triggers are important functions of client, like running single step of state sync or checking if we can produce a block.

It is called before processing Actix message and also in `schedule_triggers`. This is to ensure all triggers enjoy higher priority than any actix message. (describe the actix bug)

Returns the delay when the next time check_triggers should be called, which is min(time until the closest trigger, 1 second)